src/components/CaseViewer.tsx

```diff
@@ -52,32 +53,72 @@ class Viewer extends React.Component<ViewerProps, ViewerState> {
     this.handleSeriesSelection = this.handleSeriesSelection.bind(this)
   }
 
-  componentDidMount (): void {
+  async componentDidMount (): Promise<void> {
+    this.setState(state => ({ isLoading: true }))
+
+    const instancesMetadataArray = await this.fetchInstancesMetadataArray()
+
+    let selectedSeriesInstanceUID = ''
```

Comment:
```suggestion
    let selectedSeriesInstanceUID = undefined
```

See [this comment](https://github.com/MGHComputationalPathology/slim/pull/25#discussion_r666149795)

Reply:
This would allow the TypeScript compiler to catch the problem. Setting the value to an empty string will just propagate the problem.

Reply:
done in https://github.com/MGHComputationalPathology/slim/pull/25/commits/1fba907d84eddde77ccd787686a243b3af4061a2